gems/administrate/CVE-2016-3098.yml

```diff
@@ -0,0 +1,16 @@
+---
+gem: administrate
+cve: 2016-3098
+title: Cross-site request forgery (CSRF) vulnerability in administrate gem
+date: 2016-04-01
+
+description: >-
+  `Administrate::ApplicationController` actions didn't have CSRF
+  protection. Remote attackers can hijack user's sessions and use any
+  functionality that administrate exposes on their behalf.
+
+unaffected_versions:
```

Comment:
Not needed here, as `patched_versions` covers this. Just remove `unaffected_versions` part.